hxrsnd/sndmotor.py

```diff
@@ -4,166 +4,26 @@
-        """
-        return self.move(rel_position + self.position, *args, **kwargs)
-
-    @stop_on_keyboardinterrupt
-    def mv(self, position, *args, **kwargs):
-        """
-        Move to a specified position, optionally waiting for motion to
-        complete. 
-
-        Parameters
-        ----------
-        position
-            Position to move to.
-
-        wait : bool, optional
-            Wait for the motor to complete the motion.
-
-        moved_cb : callable
-            Call this callback when movement has finished. This callback must
-            accept one keyword argument: 'obj' which will be set to this
-            positioner instance.
-
-        timeout : float, optional
-            Maximum time to wait for the motion. If None, the default timeout
-            for this positioner is used.
-
-        Returns
-        -------
-        status : MoveStatus        
-            Status object for the move.
-        """
-        return self.move(position, *args, **kwargs)
-
-    def mvr(self, rel_position, *args, **kwargs):
-        """
-        Move relative to the current position, optionally waiting for motion to
-        complete. Alias for self.mv(rel_position + self.position).
-
-        Parameters
-        ----------
-        rel_position
-            Relative position to move to.
-
-        wait : bool, optional
-            Wait for the motor to complete the motion.
-
-        moved_cb : callable
-            Call this callback when movement has finished. This callback must
-            accept one keyword argument: 'obj' which will be set to this
-            positioner instance.
-
-        timeout : float, optional
-            Maximum time to wait for the motion. If None, the default timeout
-            for this positioner is used.
-
-        Returns
-        -------
-        status : MoveStatus        
-            Status object for the move.
-        """
-        return self.mv(rel_position + self.position, *args, **kwargs)
-
-    def __call__(self, position, *args, **kwargs):
-        """
-        Moves the motor to the inputted position. Alias for self.mv(position).
-
-        Parameters
-        ----------
-        position
-            Position to move to.
-
-        wait : bool, optional
-            Wait for the motor to complete the motion.
-
-        moved_cb : callable
-            Call this callback when movement has finished. This callback must
-            accept one keyword argument: 'obj' which will be set to this
-            positioner instance.
-
-        timeout : float, optional
-            Maximum time to wait for the motion. If None, the default timeout
-            for this positioner is used.
-
-        Returns
-        -------
-        status : MoveStatus 
-            Status object for the move.
-        """
-        return self.mv(position, *args, **kwargs)
+    pass
 
 
-class SndEpicsMotor(SndMotor, EpicsMotor):
+class SndEpicsMotor(PCDSMotorBase, SndMotor):
```

Comment:
Does this ordering of inheritance still give sublasses the `desc` attribute?

Reply:
Seems to work:
```
In [1]: snd.desc
Out[1]: 'snd'

In [2]: snd.name
Out[2]: 'snd'

In [3]: snd.E1.desc
Out[3]: 'Delay Energy'

In [4]: snd.t1.chi1.desc
Out[4]: 'T1 CHI1'

In [5]: 
```